tmp_spec.rb

```diff
@@ -0,0 +1,38 @@
+require("rails_helper")
```

Comment:
This file is introduced as the only change in a commit called `Fix Performance/HashEachMethods offenses`.  Either the commit should be re-written to explain what this test is, or we shoudl drop the commit, and this test.